README.md

```diff
@@ -35,40 +36,59 @@ which maintains the project and offers professional services around it.
 - Small memory footprint and fast execution
 - Domain-driven design architecture, high-quality code and test coverage
 
-## Install
-Before proceeding, install the following prerequisites:
+## Prerequisites
+
+The following are needed to run Mainflux:
+
+- [Docker](https://docs.docker.com/install/) **(version 18.09)**
+- [Docker compose](https://docs.docker.com/compose/install/) **(version 1.24.1)****(template version 3.7)**
```

Comment:
Are we sure that `**(template version 3.7)**` will render bold, or is it missing one space before?

Reply:
Also - no need really for these versions to be bolded.

For docker-copose I would put something like `(version 1.24.1, template version 3.7 for Dockerfiles)` - or even better I would ommit any infirmation about Dockerfile template version for now - this is not a prerequisit to running the code, but to a development (and this should not ne noted in README, but rather maybe in dev'd guide).

Reply:
I remove the bolding around versions and move the template version to the dev guide